fixtures/build5.test.yaml

```diff
@@ -0,0 +1,26 @@
+my-flavour:
+  triggers:
+    push:
+      branches:
+        - dev
+        - main
+  build_args: 
+    APP: foo
+    App2: lol
+    PAT: ${{env.PAT}}
```

Comment:
I would test the following expressions too:

```yaml
PAT: ${{ env.PAT }}
PAT: "${{env.PAT}}"
PAT: '${{env.PAT}}'
PAT: "${{ env.PAT }}"
...
```

Reply:
solved and tested.

Reply:
I miss a test with single quotes